runtime/compiler/control/J9Options.cpp

```diff
@@ -2200,7 +2210,14 @@ J9::Options::setupJITServerOptions()
                (unsigned long long) compInfo->getPersistentInfo()->getClientUID());
          }
       }
-
+   // Set two threshold very high and high to default values if the thresholds
+   // are greater than the number of usable compilation threads or if the high threshold is greater than very high.  
+   if (getNumUsableCompilationThreads() < _veryHighActiveThreadThreshold || getNumUsableCompilationThreads() < _highActiveThreadThreshold
+      || _highActiveThreadThreshold > _veryHighActiveThreadThreshold)
+      {
+         _veryHighActiveThreadThreshold = getNumUsableCompilationThreads() * 0.9;
+         _highActiveThreadThreshold = -1;
```

Comment:
Who sets `_highActiveThreadThreshold`? It appears to me that this threshold will always be -1